### PR TITLE
[TINY] RevEng: Throw error if user attempts to run Scaffold-DbContext in a VB project.

### PR DESCRIPTION
**Pull request by lajones**
Fix for #3417.

### CODE REVIEW

File: src/EntityFramework.Commands/tools/EntityFramework.psm1 [hunk 1]
```diff
@@ -409,6 +409,9 @@ function Scaffold-DbContext {
     $values = ProcessCommonParameters $StartupProject $Project
     $dteStartupProject = $values.StartupProject
     $dteProject = $values.Project
+    if ([string]::Compare($dteProject.Kind.ToString(), "{F184B08F-C81C-45F6-A57F-5ABD9991F28F}", $True) -eq 0) {
+        throw "Project $Project is a Visual Basic project which is not yet supported by Scaffold-DbContext."
+    }
```
Comment (rowanmiller) [inline src/EntityFramework.Commands/tools/EntityFramework.psm1]: Does this cover all VB.NET projects, or just class library?

**Comment by bricelam** [comment]
Why doesn't it just spit out C# code? The project models are almost identical...

File: src/EntityFramework.Commands/tools/EntityFramework.psm1 (hunk 1 above)
Comment (ErikEJ) [inline src/EntityFramework.Commands/tools/EntityFramework.psm1]: I would say all relevant project types: http://www.mztools.com/articles/2008/mz2008017.aspx

Comment (bricelam) [inline src/EntityFramework.Commands/tools/EntityFramework.psm1]: In EF6, we detected using `project.CodeModel.Language == CodeModelLanguageConstants.vsCMLanguageVB`. I believe this caught all projects including Web Sites.

**Comment by bricelam** [comment]
I don't like the idea of blocking users from doing this. In Migrations, they can override the `MigrationsCodeGenerator` design-time service with a VB-specific one and everything should work. It would be nice to have the same seam in RevEng.

**Comment by bricelam** [comment]
I believe we said in the past that we wouldn't necessarily support VB, but we also wouldn't explicitly block it either.

File: src/EntityFramework.Commands/tools/EntityFramework.psm1 (hunk 1 above)
Comment (lajones) [inline src/EntityFramework.Commands/tools/EntityFramework.psm1]: I'll test a few more project types, but the above code was taken from `VsUtils` in EF6.

**Comment by lajones** [comment]
@bricelam As mentioned in the issue #3417 I'm happy to just leave the code as is - in which case it does successfully output C# code into the VB project. But @rowanmiller is in favor of the error message...

**Comment by rowanmiller** [comment]
Oh, right, yeah I missed that we would be blocking someone from implementing support themselves... yeah we should not block that.

**Comment by rowanmiller** [comment]
Ok, yeah... just C# is fine

**Comment by lajones** [comment]
OK. I'll close this and update the issue to say we're leaving it as is.

**Pull request closed without merging**